### ISSUE TITLE
adding more SDF primitives

### ISSUE THREAD

**Issue by mutablehurdle** [portnov/sverchok-extra#44]
I have written a few SDF primitives for my own usage and just realized that SDF had been incorporated into Sverchok-Extra. I may not be familiar enough with how the Sverchok node tree is structure, is there a way to add my custom nodes to the list of primitives?

**Comment by portnov** [comment]
Hello.
Create github's pull request, I'll review it and most probably merge.

**Comment by mutablehurdle** [comment]
Pull request created. I updated parts of the sdf module itself to make it work, so I guess merging would require updating the dependency to include that branch? Sorry I am not well versed in git version control and dependencies

**Comment by portnov** [comment]
Yes, in order for this to work, we will have to wait until https://github.com/fogleman/sdf/pull/31 is merged. I subscribed there :)
